src/windows/WindowMessages.hpp

```diff
@@ -65,6 +65,8 @@ enum eWmUserMsgs
 	WM_NOTIFMANAGERCALLBACK,
 	WM_POSTINIT,
 	WM_RECREATEMEMBERLIST,
+	WM_CLOSEBYPASSTRAY,
 
 	WM_UPDATETEXTSIZE = WM_APP, // used by the MessageEditor
+	WM_RESTORE,
```

Comment:
May want to call it something like `WM_RESTOREAPP`, because more than once did I confuse this for an actual Windows window message